src/pc/1.18/ChunkColumn.js

```diff
@@ -14,9 +15,11 @@ module.exports = (Block, mcData) => {
       this.minY = options?.minY ?? 0
       this.worldHeight = options?.worldHeight ?? constants.CHUNK_HEIGHT
       this.numSections = this.worldHeight >> 4
+      this.maxBitsPerBlock = neededBits(Object.values(mcData.blocks).reduce((high, block) => Math.max(high, block.maxStateId), 0))
+      this.maxBitsPerBiome = neededBits(Object.values(mcData.biomes).length)
```

Comment:
Was this line left over from testing?

Reply:
No. This is needed for proper direct palette parsing

Reply:
One of those lines is not needed though, right? It seems the rhs has no side effects and the second statement overwrites the computed value from the first.

Reply:
Ah, perBlock and perBiome, nvm I see it now

Reply:
One is bits per block. The other is bits per biome